bindings/c/include/opendal.h

```diff
@@ -79,6 +79,38 @@ typedef enum opendal_code {
   OPENDAL_IS_SAME_FILE,
 } opendal_code;
 
+/*
+ BlockingOperator is the entry for all public blocking APIs.
+
+ Read [`concepts`][docs::concepts] for know more about [`Operator`].
+
+ # Examples
+
+ Read more backend init examples in [`services`]
+
+ ```
+ # use anyhow::Result;
+ use opendal::services::Fs;
+ use opendal::BlockingOperator;
+ use opendal::Operator;
+ #[tokio::main]
+ async fn main() -> Result<()> {
+     // Create fs backend builder.
+     let mut builder = Fs::default();
+     // Set the root for fs, all operations will happen under this root.
+     //
+     // NOTE: the root must be absolute path.
+     builder.root("/tmp");
+
+     // Build an `BlockingOperator` to start operating the storage.
+     let _: BlockingOperator = Operator::new(builder)?.finish().blocking();
+
+     Ok(())
```

Comment:
We may donnot need those comments. It's for rust.

Reply:
it's automatically generated

Reply:
😂😂😂Well...